### PR TITLE
Easyprocess feedstock

### PR DESCRIPTION
**Pull request by ReimarBauer**
easyprocess is a dependency by pyvirtualdisplay

### CODE REVIEW

File: recipes/easyprocess-feedstock/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - python
+    
+
+test:
+  imports:
+    - easyprocess
+
+about:
+  home: https://github.com/ponty/EasyProcess
+  license: BSD
```
Comment (jakirkham) [inline recipes/easyprocess-feedstock/meta.yaml]: Please include version. For example, this appears to be `BSD 2-Clause`.

File: recipes/easyprocess-feedstock/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+source:
+  fn: easyprocess-{{ version }}.tar.gz
+  url:  https://github.com/ponty/EasyProcess/archive/{{version}}.tar.gz
+  sha256: 82f0323a3e4ecdb1271392b81bfd03bcd8639b9bc6fb638845aa2c53a0dea7e9
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: True
```
Comment (jakirkham) [inline recipes/easyprocess-feedstock/meta.yaml]: Why is this necessary?

File: recipes/pyvirtualdisplay-feedstock/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+source:
+  fn: pyvirtualdisplay-{{ version }}.tar.gz
+  url:  https://github.com/ponty/PyVirtualDisplay/archive/{{ version}}.tar.gz
+  sha256: 2c6bf006de7b1e950fae264d337c4344540e9f96d76e46f763fbb53e18798317
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: True
```
Comment (jakirkham) [inline recipes/pyvirtualdisplay-feedstock/meta.yaml]: Why is this necessary?

File: recipes/easyprocess-feedstock/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  name: easyprocess
+  version: {{ version }}
+
+source:
+  fn: easyprocess-{{ version }}.tar.gz
+  url:  https://github.com/ponty/EasyProcess/archive/{{version}}.tar.gz
+  sha256: 82f0323a3e4ecdb1271392b81bfd03bcd8639b9bc6fb638845aa2c53a0dea7e9
+
+build:
+  skip: true  # [win]
```
Comment (jakirkham) [inline recipes/easyprocess-feedstock/meta.yaml]: What is the reason for skipping Windows?

File: recipes/pyvirtualdisplay-feedstock/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  name: pyvirtualdisplay
+  version: {{ version }}
+
+source:
+  fn: pyvirtualdisplay-{{ version }}.tar.gz
+  url:  https://github.com/ponty/PyVirtualDisplay/archive/{{ version}}.tar.gz
+  sha256: 2c6bf006de7b1e950fae264d337c4344540e9f96d76e46f763fbb53e18798317
+
+build:
+  skip: true  # [win]
```
Comment (jakirkham) [inline recipes/pyvirtualdisplay-feedstock/meta.yaml]: What is the reason for skipping Windows?

File: recipes/pyvirtualdisplay-feedstock/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    - python
+    - easyprocess
+
+test:
+  imports:
+    - pyvirtualdisplay
+
+about:
+  home: https://github.com/ponty/pyvirtualdisplay
+  license: BSD
```
Comment (jakirkham) [inline recipes/pyvirtualdisplay-feedstock/meta.yaml]: Please include version. For example, this appears to be `BSD 2-Clause`.

**Comment by jakirkham** [comment]
FYI directory names should match package names.

**Comment by jakirkham** [comment]
Mostly good. Couple areas to adjust. Otherwise should be good to go.

File: recipes/easyprocess-feedstock/meta.yaml (hunk 2 above)
Comment (ReimarBauer) [inline recipes/easyprocess-feedstock/meta.yaml]: hmm, I think it isn't, Left over from a copy und paste recipe.

File: recipes/easyprocess-feedstock/meta.yaml (hunk 4 above)
Comment (ReimarBauer) [inline recipes/easyprocess-feedstock/meta.yaml]: hmm, removed

File: recipes/pyvirtualdisplay-feedstock/meta.yaml (hunk 5 above)
Comment (ReimarBauer) [inline recipes/pyvirtualdisplay-feedstock/meta.yaml]: pyvirtualdisplay uses as back-ends: Xvfb, Xephyr, Xvnc 
those are not available on windows

File: recipes/pyvirtualdisplay-feedstock/meta.yaml (hunk 6 above)
Comment (ReimarBauer) [inline recipes/pyvirtualdisplay-feedstock/meta.yaml]: ok

**Comment by jakirkham** [comment]
Thanks @ReimarBauer.
